### PR TITLE
Implements Maim / Maiming projectiles.

### PR DESCRIPTION
**Pull request by chaoko99**
Should get rid of some snowflake code.

### CODE REVIEW

**Comment by chaoko99** [comment]
For the record: I opened this PR for the sake of discussion. I know it doesn't work yet.

File: code/modules/organs/organ_external.dm [hunk 1]
```diff
@@ -296,21 +298,29 @@
...
+							gibs_traditionally = FALSE
+
+					else
+						if(W.w_class >= w_class)
+							edge_eligible = 1
 				else
 					edge_eligible = 1
 
-			if(edge_eligible && brute >= max_damage / DROPLIMB_THRESHOLD_EDGE && prob(brute))
+			if(gibs_traditionally && edge_eligible && brute >= max_damage / DROPLIMB_THRESHOLD_EDGE && prob(brute))
```
Comment (Lohikar) [inline code/modules/organs/organ_external.dm]: You could just change this to:
```dm
if (gibs_traditionally)
    if (edge_eligible && brute >= max_damage / DROPLIMB_THRESHOLD_EDGE && prob(brute))
        // etc
```

Comment (chaoko99) [reply code/modules/organs/organ_external.dm]: Resolved.

File: code/modules/organs/organ_external.dm [hunk 2]
```diff
@@ -310,18 +310,19 @@
 
 					else
 						if(W.w_class >= w_class)
-							edge_eligible = 1
+						edge_eligible = 1
```
Comment (Lohikar) [inline code/modules/organs/organ_external.dm]: This change looks unintentional.

Comment (chaoko99) [reply code/modules/organs/organ_external.dm]: Resolved.

Comment (chaoko99) [reply code/modules/organs/organ_external.dm]: Resolved.

**Comment by chaoko99** [comment]
Merge me daddy

**Comment by printer16** [comment]
Conflicts now.

**Comment by chaoko99** [comment]
Which I would hazard to guess is why we look at these things the same month they are posted.

**Comment by chaoko99** [comment]
Resolved.

**Pull request merged**